src/ontology/edits/oeo-physical.omn

```diff
@@ -3254,6 +3254,31 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/385"@en,
         <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000420
     
     
+Class: OEO_00020037
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Radiation is the process of emitting or transmitting energy in the form of waves or particles through a spatial region or a material entity.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "Adaptation of https://en.wikipedia.org/wiki/Radiation",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000119> "Adaptation of https://en.wikipedia.org/w/index.php?title=Radiation&oldid=986678480",
```
I suggest using here the permanent link to the current version of the Wikipedia article as any Wikipedia article is subject for constant change.